build.gradle

```diff
@@ -31,6 +31,7 @@ repositories {
 }
 
 dependencies {
+    implementation 'org.junit.jupiter:junit-jupiter:5.4.2'
```

Comment:
The JUnit stuff we need should already be imported below. Remove this line and resync gradle, the tests should still work.
```suggestion
```